pkg/framer/property_values_for_time_range.go

```diff
@@ -2,20 +2,21 @@ package framer
 
 import (
```

Comment:
I think the content of this got switched with pkg/framer/property_values_for_time_range_batch.go